example/image-classification/train_imagenet.py

```diff
@@ -52,7 +52,11 @@
 
 # network
 import importlib
-net = importlib.import_module('symbol_' + args.network).get_symbol(args.num_classes)
+if args.network == 'inception-resnet-v2':
```

Comment:
@zhreshold  Thanks!  It's very very very helpful to me!!! I will update the code! Could you help to review later